src/stories/OptionRow.stories.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import OptionRow from '../components/OptionRow';
 import * as Expensicons from '../components/Icon/Expensicons';
+import OnyxProvider from '../components/OnyxProvider';
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Just checking about this, why do we move it here? It's kind of odd to have a comment after the code instead of before, but maybe these are new style rules.

Reply:
When I wrapped the OptionRow in the OnyxProvider I couldn't put the 'eslint-disable-next-line' comment in the jsx, so I just disable it for the whole file and I put it at the top because that made sense to me.